src/GitHubExtension/DataManager/GitHubDataManager.cs

```diff
@@ -514,6 +536,36 @@ private async Task UpdatePullRequestsAsync(Repository repository, Octokit.GitHub
         PullRequest.DeleteLastObservedBefore(DataStore, repository.Id, DateTime.UtcNow - LastObservedDeleteSpan);
     }
 
+    private void ProcessReview(PullRequest pullRequest, Octokit.PullRequestReview octoReview)
+    {
+        // Skip reviews that are stale.
+        if ((DateTime.Now - octoReview.SubmittedAt) > ReviewStaleTime)
+        {
+            return;
+        }
+
+        // For creating review notifications, must first determine if the review has changed.
+        var existingReview = Review.GetByInternalId(DataStore, octoReview.Id);
+
+        // Add/update the review record.
+        var newReview = Review.GetOrCreateByOctokitReview(DataStore, octoReview, pullRequest.Id);
+
+        // Ignore comments or pending state.

```

Comment:
Should this be a setting? I can imagine wanting to get notifications for comments.

Reply:
We plan on making more granular notification settings for later as we expand the types. Since we don't have that setting granularity yet we are erring on the side of caution and not being too spammy with the notifications. They're aimed at notifications the developer would very likely be interested in having enabled and so it is tied to the one notification setting. When we have more granular settings with different defaults we will add comments, PR assignments, PRs created by other people, etc.